tests/framework/db/oci/CommandTest.php

```
@@ -33,14 +38,87 @@ public function testAutoQuoting(): void
+            SQL
+        )->execute();
+
+        self::assertNotEmpty(
+            $db->getSchema()->getLastInsertID($sequenceName),
+            'CURRVAL of the IDENTITY system sequence must reflect the inserted row.',
+        );
+    }
+
+    public function testGetLastInsertIDReturnsValueViaLegacyTriggerBackedSequence(): void
+    {
+        $db = $this->getConnection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)